geti_sdk/annotation_readers/geti_annotation_reader.py

```diff
@@ -160,6 +161,18 @@ def get_data(
                             label_name=label_dict["name"]
                         )
             new_annotations.append(annotation_object)
+            if "anomal" in annotation_object.labels[0].name.lower():
```

Comment:
Why don't you just check `annotation_object.labels[0].name.lower() == "anomalous"`? I believe that has been a constant throughout all older versions of Geti, the name of the anomalous label has not changed.

Reply:
We should not rely on the label name. Unfortunately, we do not save the 'is_anomalous' label flag.